core/new-gui/src/app/workspace/component/workflow-editor/workflow-editor.component.ts

```diff
@@ -727,7 +785,12 @@ export class WorkflowEditorComponent implements AfterViewInit {
       )
       .pipe(untilDestroyed(this))
       .subscribe(elementView => {
-        this.workflowActionService.deleteOperator(elementView.model.id.toString());
+        if (this.workflowActionService.getTexeraGraph().hasOperator(elementView.model.id.toString())) {
+          this.workflowActionService.deleteOperator(elementView.model.id.toString());
+        }
+        if (this.workflowActionService.getTexeraGraph().hasCommentBox(elementView.model.id.toString())) {
+          this.workflowActionService.deleteCommentBox(elementView.model.id.toString());
+        }
```

Comment:
I am hesitating about whether we should do the existence check inside the delete methods. Was this a bug before?

Reply:
It's the convention for other element checks, I think we can have it inside the delete method but it would have a different syntax.